crates/daphne/src/vdaf/prio3_latest.rs

```diff
@@ -0,0 +1,816 @@
+// Copyright (c) 2022 Cloudflare, Inc. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+
+//! Parameters for the [Prio3 VDAF](https://datatracker.ietf.org/doc/draft-patton-cfrg-vdaf/).
+
+use crate::{
+    fatal_error,
+    messages::{taskprov::VDAF_TYPE_PRIO3_SUM_VEC_FIELD64_MULTIPROOF_HMAC_SHA256_AES128, TaskId},
+    vdaf::{VdafError, VdafVerifyKey},
+    DapAggregateResult, DapMeasurement, Prio3Config, VdafAggregateShare, VdafPrepShare,
+    VdafPrepState,
+};
+use prio_latest::{
+    codec::{Encode, ParameterizedDecode},
+    field::Field64,
+    flp::{
+        gadgets::{Mul, ParallelSum},
+        types::SumVec,
+        Type,
+    },
+    vdaf::{
+        prio3::{Prio3, Prio3InputShare, Prio3PrepareShare, Prio3PrepareState, Prio3PublicShare},
+        xof::Xof,
+        xof::XofHmacSha256Aes128,
+        Aggregator, Client, Collector, PrepareTransition, Vdaf,
+    },
+};
+
+const ERR_FIELD_TYPE: &str = "unexpected field type for step or message";
+const CTX_STRING: &[u8] = "dap-13".as_bytes();
+const VERSION: u8 = 8;
+
+type Prio3LatestSumVecField64MultiproofHmacSha256Aes128 =
+    Prio3<SumVec<Field64, ParallelSum<Field64, Mul<Field64>>>, XofHmacSha256Aes128, 32>;
+
+fn new_prio3_latest_sum_vec_field64_multiproof_hmac_sha256_aes128(
+    bits: usize,
+    length: usize,
+    chunk_length: usize,
+    num_proofs: u8,
+) -> Result<Prio3LatestSumVecField64MultiproofHmacSha256Aes128, VdafError> {
+    Prio3::new(
+        2,
+        num_proofs,
+        VDAF_TYPE_PRIO3_SUM_VEC_FIELD64_MULTIPROOF_HMAC_SHA256_AES128,
+        SumVec::new(bits, length, chunk_length).map_err(|e| {
+            VdafError::Dap(fatal_error!(
+                err = ?e,
+                "failed to create sum vec from bits({bits}), length({length}), chunk_length({chunk_length})"
+            ))
+        })?,
+    )
+    .map_err(|e| VdafError::Dap(fatal_error!(err = ?e, "failed to create prio3")))
+}
```

Comment:
Let's remove `Prio3LatestSumVecField64MultiproofHmacSha256Aes128`, since we don't need it. Note that we should expect the test vectors to fail.

Reply:
IIUC we do in fact still need this.